table/destination.go

```diff
@@ -500,6 +502,14 @@ func (dest *Destination) computeKnownBestPath() (*Path, BestPathReason, error) {
 		return nil, BPR_UNKNOWN, nil
 	}
 
+	if SelectionOptions.DisableBestPathSelection {
+		log.WithFields(log.Fields{
+			"Topic": "Table",
+		}).Debug("computeKnownBestPath skipped")
+		// Note: The older path gets preference, because "newPathList" was
+		// appended after "knownPathList".
+		return dest.knownPathList[0], BPR_DISABLED, nil
+	}
```

Comment:
I think that the best path selection is disabled, nothing should be sent to peers because there is no best path. This is for a use-case that gobgp receives routes from peers and sends nothing, usually called, a route-collector.